### DISCUSSION TITLE
读《当代中国政府与政治》 | /posts/ccgp/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#617] [Announcements]
https://yuanfan.rbind.io/posts/ccgp/

**Comment by botsp** [comment]
最近听邱会作的回忆录，有段时期存在"军委办事组"这样一个机构，也负责政务，按照他的说法几乎等同"军政府"了。
他有段描述特别有意思:

*在总理领导的政治局的工作中，军委办事组是中流砥柱。我不是说我们这些人有多大本事，而是说，党给了你这个位子，你就为党的根本利益在努力工作。有些人号称和「四人帮」作了斗争，其实，他们和「四人帮」的所谓「斗争」是在毛主席逝世以后，是在「四人帮」被打倒以后，是在否定文化大革命以后，那算什么斗争呢？！我们和「四人帮」（那时还不这么叫）的斗争，是毛主席健在的时候，是江青他们在台上是领导人的时候，是肯定文革的时候，那个时候，有些自诩和江青斗争的人，正含着热泪向主席请罪呢。*

**Reply by earfanfan** [reply]
这本书里专门有一个章节讲“军事系统”。在第172页提到军队系统的功能之一是，维护中国共产党的执政地位，

>在建军之初，党就把军队当作保护自己的重要力量，并视为夺取政权、建立国家的主要工具。在中国国家政权的建构过程中，党始终居于核心领导地位。此后，军队也一直致力于“为中国共产党巩固执政地位提供重要的力量保证”。当党的统治体系发生整体性紊乱时，军队就成为及时的“稳定器”，如在“文化大革命”期间，党政官僚体系被打乱而不能正常运作时，军队进行“三支两军”，组建革命委员会，掌握了地方党政机构的实际权力，为党政机构的重建提供了过渡性的安排。1989年政治风波中，军队再次在关键时刻挽救了党，恢复了国家社会秩序和政治稳定，使中国的政治发展走上一条不同于苏联的道路。在当前中国政治体制改革过程中，军队仍是决定中国政治未来走向的主要因素之一。简言之，军队在关键时刻扮演的党的护持者角色，使掌握政治权力的共产党能按自己的战略意志建构国家，掌握、调整国家发展的方向与路线。

你摘录的这段描述在书中也有对应的内容：

>“文化大革命”时期全国实行军管，上至国务院各部委，下至县政部门，形成了军人执政的局面。

至于军委和政府的关系，也有阐述：

>根据宪法规定，国务院设立国防部，领导和管理国防建设事业。一切需要由政府负责的军事工作，经国务院作出相应决定，通过国防部或以国防部的名义组织实施。国防部在接受国务院领导的同时也接受中央军事委员会的领导。国防部部长既是中央军委成员，也是国务院行政部门的首长。在实际运作中，需要国防部办理的事宜，由总参谋部、总政治部、总后勤部、总装备部分别处置。由于“四总部”直接向中央军委负责，所以实际上中央军委“几乎总揽了我国军事统帅权和军事行政权”。在这一体制下，国防部的军事行政权一直处于虚置状态,主要负责军事外交，参与国防动员协调。原国务院副总理陈慕华曾说，“军事一向是由军委管，国防部是虚的，这样做，大家习惯了”。作为国家行政机关内设机构的国防部与中央军委及“四总部”之间的这种权力关系.体现了最高军事机构与国家行政机构之间的平行关系，而不是隶属关系。

**Reply by botsp** [reply]
对的，应该就是这个“三支两军”。比较特别的历史时期。
